frame/ibc/ibc-primitives/src/runtime_interface.rs

```diff
@@ -164,3 +163,16 @@ fn ss58hash(data: &[u8]) -> Vec<u8> {
 	pre_image.extend(data);
 	sp_io::hashing::blake2_256(&pre_image).to_vec()
 }
+
+#[derive(Debug)]
+pub struct Blake2HostFunctionsHasher;
```

Comment:
We can use the `sp_runtime::traits::BlakeTwo256`, it already implements this.

Reply:
good catch